roles/defaults/defaults/main.yml

```diff
@@ -167,3 +167,8 @@ atmosphere_images:
   skopeo: quay.io/skopeo/stable:latest
   tempest_run_tests: us-docker.pkg.dev/vexxhost-infra/openstack/tempest:30.1.0-4
   vector: docker.io/timberio/vector:0.27.0-debian
+  manila_db_sync: quay.io/vexxhost/manila:zed
```

Comment:
Can you please put these in alphabetical order?

Reply:
ok